Add missing docstrings for ONNX converter API. Fixes #67393 (#67640)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69550
* #69549
* #69548
* #69547
* #69546
* #69544
* #68493
* #68492
* #68491
* #68490
* __->__ #68489

Differential Revision: [D32483783](https://our.internmc.facebook.com/intern/diff/D32483783)

BowenBao committed:
Add missing docstrings for ONNX converter API. Fixes #67393 (#67640)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483783).

BowenBao committed:
Update on "Add missing docstrings for ONNX converter API. Fixes #67393 (#67640)"


Differential Revision: [D32483783](https://our.internmc.facebook.com/intern/diff/D32483783)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483783).

BowenBao committed:
Update on "Add missing docstrings for ONNX converter API. Fixes #67393 (#67640)"


Differential Revision: [D32483783](https://our.internmc.facebook.com/intern/diff/D32483783)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32483783).
